routers/router.go

```diff
@@ -11,6 +11,8 @@ func init() {
 	//
 	web.Router("/", &controllers.ResultController{}, "get:List")
 	web.Router("/results", &controllers.ResultController{}, "post:Create")
+	web.Router("/results/:id", &controllers.ResultController{}, "get:Show")
```

Comment:
WE can validate the id in the route level
```suggestion
	web.Router("/results/:id:int", &controllers.ResultController{}, "get:Show")
```

Reply:
updated in [57121c1](https://github.com/carryall/go-google-scraper-challenge/pull/72/commits/57121c19ad3a0cdb228dc2a8bde8209411e433a7)